applications/vessel-history/src/features/event-filters/filters.hooks.ts

```diff
@@ -10,17 +11,26 @@ export const useApplyFiltersConnect = () => {
   const filters = useSelector(selectFilters)
 
   const setFilter = (
+    tab: 'MAP' | 'ACTIVITY',
     filter: availableEventFilters,
     value: boolean
   ) => {
     const newFilters = {
       ...filters,
       [filter]: value,
     }
+
     dispatch(updateFilters(newFilters))
+
+    uaEvent({
+      category: 'Vessel Detail ACTIVITY or MAP Tab',
+      action: 'Click Filter Icon - Event type',
+      label: JSON.stringify({ [filter]: value, tab: tab })
+    })
   }
```

Comment:
Same as https://github.com/GlobalFishingWatch/frontend/pull/932/files#r711103276, I would not attach this to the hook but the component